src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java

```diff
@@ -28,31 +41,21 @@ public void initTestValueProvider() {
         WebDriverManager.chromedriver().setup();
     }
 
+    @Parameters("browser")
     @BeforeClass(description = "Init ChromeDriver.")
-    protected void initDriver(ITestContext context) {
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
-//        options.addArguments("--headless");
-        driver = new ChromeDriver(options);
-        context.setAttribute("myDriver", driver);
+    protected void initDriver(String browser, ITestContext context) throws InterruptedException {
+        driver = browsers.setUpBrowser(browser);
         driver.manage().window().maximize();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
         driver.get(valueProvider.getBaseUiUrl());
-        driver.findElement(By.id("details-button")).click();
-        driver.findElement(By.id("proceed-link")).click();
+        checkErrorPage(browser);
     }
 
+    @Parameters("browser")
     @AfterClass
     public void closeDriver() {
         if (driver != null) {
             driver.close();
         }
```

Comment:
change to `driver.quit()`, as after tests we need close window and also kill process in Task Manager. 
`driver.quit()` is more suitable in this case

Reply:
done